fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -323,6 +328,49 @@ class DefaultSessionControlController(
         }
     }
 
+    override fun handleOpenInBackgroundTabClicked(topSite: TopSite, position: Int) {
```

Comment:
This seems to duplicate the code from [handleSelectTopSite](https://searchfox.org/mozilla-mobile/rev/2d9dd7b862d87ac539a52b9b0d99160b61745dae/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt#391-432) with the only difference being the `selectTab = false` parameter of `addTabUseCase`.
If the goal is to keep the two methods in sync then to prevent future refactorings only modifying one behavior we should look into reusing the exesting code. 
I propose adding a new parameter to the `handleSelectTopSite` methods from the Controller, maybe `selectTab` with the two methods from the Interaction calling this with either `true` or `false`.